config/manager/manager.yaml

```diff
@@ -33,6 +33,11 @@ spec:
         name: manager
         securityContext:
           allowPrivilegeEscalation: false
+          privileged: false
+          runAsNonRoot: true
```

Comment:
Just want to confirm that this worked in your testing. IIRC we had an issue in our server deployments where we specified `runAsNonRoot` but didn't specify the user it *should* run as, and so the deployment errored.

Reply:
This did work in my testing but I'll probe at it a little bit to be extra sure that it's doing what I expect.

Reply:
Just took a closer look at this -- Our Dockerfile uses the `gcr.io/distroless/static:nonroot` image and specifies the 65532 user (which is the default nonroot user).